public/css/common.less

```diff
@@ -151,17 +159,25 @@ div.show-more {
     // Icinga Web 2 override
     .spinner {
       height: 100%;
-      padding-right: 0;
+    }
+
+    .control-button {
+      margin: 0;
+    }
+
+    .link-button {
+      min-width: 2em;
+      text-align: center;
+      .rounded-corners();
+
+      &:hover {
+        background: @gray-lighter;
+      }
     }
   }
 
   .show-more {
     margin-top: .25em;
-    margin-left: 6em;
```

Comment:
This leads to less differentiation between the quick actions and the show all link:
![Screenshot from 2021-06-09 17-39-07](https://user-images.githubusercontent.com/16668527/121385796-b7e58500-c949-11eb-8a99-85155d52865b.png)

Some gap is of advantage here IMHO. It also looks weird because below there's empty space anyway.

Reply:
Can you check, what version you checked out? On the screenshot it doesn't look like it was intended. This is what the controls are supposed to look.

![Screen Shot 2021-06-30 at 14 57 59](https://user-images.githubusercontent.com/6977434/123964495-c1e02e00-d9b3-11eb-840e-bdd326edba12.png)

Reply:
This way it's also tidier and clearer, which elements belong where. Heights match the row grid and I'd say it's the same height in 80% of cases as the right floating "Show More" link.

Reply:
I'm on de6e6d65 and it's still the case. Looking at your screen, please remember the wide screen layout. I don't want a useless break there on my wide screen :wink: 
![Screenshot from 2021-07-01 09-48-49](https://user-images.githubusercontent.com/16668527/124086659-91e76800-da51-11eb-950e-b386e1f21be0.png)

Reply:
Smart. I'll make specific style for `.wide-layout`.

Reply:
In non-wide layout it still looks this way:
![Screenshot from 2021-07-13 12-31-28](https://user-images.githubusercontent.com/16668527/125437431-4022d1a4-b783-4580-aff1-c6eca51e2e34.png)